README.md

```diff
@@ -385,7 +411,7 @@ This is especially helpful if you are developing against the tool and want to qu
 Tests are included under the `/cmd` directory. There is a mixture of unit tests as well as CLI interface tests that verify certain flags are required, error messages are returned as expected when invalid input is provided, etc.
 
 ```
-go test -v ./cmd
+go test -v ./cmd/git-xargs
```

Comment:
There's no such folder now... You probably want to use the triple dot to run tests in all packages. Oh, and if tests take > 10 min, add `-timeout XXX`.

Reply:
Roger - thanks for catching that. Tests luckily take ~46 seconds right now so I will omit timeout for the time being.

Reply:
Fixed in 145d9a4 and removed some unnecessary / out of date text.